seed.d/rules.json

```diff
@@ -2,29 +2,36 @@
     {
         "rule_id": "cloud_instance_ros_evaluation|CONSUMPTION_MODEL",
         "description":"This instance is idling so consider switching to a consumption model.",
-        "reason": "Based on the host metrics set, we have detected that this {system.instance_type} instance on {system.cloud_provider.upper()} is displaying a higher amount of idling cycles within a 24h time frame. It can be potentially explained by the non-business hours. This idling state may be caused by picking an inappropriate consumption model.",
-        "resolution": "Please verify that this instance and its workload is having an appropriate consumption model for this specific workload.",
-        "condition":"CPU Score + MEM Score + I/O Score (combined)  conditioned IDLE > 30% out of 24h,  are in between 3/15 and 14/15"
+        "reason": "Based on the host metrics set, we have detected that this {system.instance_type} instance on {system.cloud_provider.upper()} region {system.cloud_region} is displaying a higher amount of idling cycles within a 24h time frame. It can be potentially explained by the non-business hours. This idling state may be caused by picking an inappropriate consumption model.\nThe specific issues identified by ROS are:\n{system.summary}",
```

Comment:
Hello @r14chandra, as ticket [RHIROS-424](https://issues.redhat.com/browse/RHIROS-424) assigned to you. Please note that here it uses `cloud_region` in rules content.
With current implementation, ROS backend don't have `region` information stored in ROS system schema. You will need to consider this as well while working on RHIROS-424.

cc @pgarciaq

Reply:
@r14chandra - One can fetch `region` information similarly like ros-backend is fetching `instance_type` information [here](https://github.com/RedHatInsights/ros-backend/blob/main/ros/processor/process_archive.py#L48).
Example -  `aws_instance_id.get('region') `

Reply:
Thansk @kgaikwad. Looking into it.